federation/cluster/federation-up.sh

```diff
@@ -22,4 +22,8 @@ KUBE_ROOT=$(readlink -m $(dirname "${BASH_SOURCE}")/../../)
 
 . ${KUBE_ROOT}/federation/cluster/common.sh
 
+if [[ -f "${KUBE_ROOT}/federation/manifests/federated-image.tag" ]]; then
```

Comment:
This has been something I've been meaning to do. We actually be very specific here:

```sh
tagfile="${KUBE_ROOT}/federation/manifests/federated-image.tag"
if [[ ! -f "$tagfile" && "${FEDERATION:-}" == "true ]];then
    echo "FATAL: FEDERATION=true, but tagfile ${tagfile} does not exist"
    exit 1
fi